docker/acr/manifest.tmpl

```diff
@@ -0,0 +1,37 @@
+image: plugins/acr:{{#if build.tag}}{{trimPrefix build.tag "v"}}{{else}}latest{{/if}}
```

Comment:
You got to change `{{trimPrefix build.tag "v"}}` to `{{trimPrefix "v" build.tag}}`, same for the other lines.